internal/server/middleware/grpc/middleware.go

```diff
@@ -324,7 +326,15 @@ func AuditUnaryInterceptor(logger *zap.Logger) grpc.UnaryServerInterceptor {
 		case *authrpc.CreateTokenRequest:
 			event = audit.NewEvent(audit.Metadata{Type: audit.Token, Action: audit.Create, Actor: actor}, r)
 		case *authrpc.DeleteAuthenticationRequest:
-			event = audit.NewEvent(audit.Metadata{Type: audit.Token, Action: audit.Delete, Actor: actor}, r)
+			auth, err := authGetter.GetAuthentication(ctx, &authrpc.GetAuthenticationRequest{Id: r.Id})
```

Comment:
we have the token id in the delete request right? i would just leave it at that and log all delete requests regardless of the method, meaning we could remove this change all together

Reply:
meaning that I know its a bit weird, but I don't think we necessarily need to differentiate between method types for the authentication token for the delete audit events here. wdyt?